tools/pubsys/src/aws/mod.rs

```diff
@@ -10,24 +9,16 @@ pub(crate) mod promote_ssm;
 pub(crate) mod publish_ami;
 pub(crate) mod ssm;
 
-/// Builds a Region from the given region name, and uses the custom endpoint from the AWS config,
-/// if specified in aws.region.REGION.endpoint.
-fn region_from_string(name: &str, aws: &AwsConfig) -> Result<Region> {
-    let maybe_endpoint = aws.region.get(name).and_then(|r| r.endpoint.clone());
-    Ok(match maybe_endpoint {
-        Some(endpoint) => Region::Custom {
-            name: name.to_string(),
-            endpoint,
-        },
-        None => name.parse().context(error::ParseRegionSnafu { name })?,
-    })
+/// Builds a Region from the given region name.
+fn region_from_string(name: &str) -> Result<Region> {
+    Ok(Region::new(name.to_owned()))
```

Comment:
If this is infallible now we should probably just return a `Region`. However, I like it this way for the purposes of the SDK migration, to minimize the diff. Maybe add a `FIXME` to rework later.

Reply:
This is a pretty quick change. Rather than a `FIXME`, should I just make it the final change once we think this PR hits all the edge cases?